04/index.html

```diff
@@ -4,27 +4,162 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <link rel="stylesheet" href="./style.css" />
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
 </head>
 <body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
-            </li>
-            <li>
-                <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
+
+    <!-- navigation website -->
+
+    <nav class="nav">
+        <ul class="menu">
+          <li class="menu__item"><a href="" class="menu__link">Start</a>
+          </li>
+          <li class="menu__item">
+              <a href="#" class="menu__link">Offer</a>
+            <ul class="menu__dropdown dropdown">
+              <li class="dropdown__item">
+                <span><a href="#" class="dropdown__link">Websites</a></span>
+                <img
+                  src="./images/icon-arrow-down.svg"
+                  alt="Arrow right"
+                  class="dropdown__arrow"
+                /> 
+                <ul class="dropdown__nested">
+                  <li class="nested__item"><a href="#" class="dropdown__link">Landing page</a></li>
+                  <li class="nested__item"><a href="#" class="dropdown__link">CMS</a></li>
+                </ul>
```

Comment:
Na wersji desktopowej menu na tym poziomie jest trudno dostępne.
Jeśli wolno przechodzimy to menu traci `:hover` i znika.
Jeśli będziesz potrzebować wskazówek jak sobie z tym poradzić to napisz o tym na Slacku - nie chce od razu cie nakierowywać ;)